### PR TITLE
ELPP-3349 Add PersonalisedCoverDownload pattern

### PR DESCRIPTION
**Pull request by thewilkybarkid**
No name's been suggested so I've just called it 'Download'. The icon's hard-coded still, could be changed later.

#813 will resolve the extra margin at the bottom of the buttons.

### CODE REVIEW

File: source/_patterns/01-molecules/components/download.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "text": "<p>You can download a poster featuring your latest paper below. It's highlighted on an imitation of an eLife journal cover.</p><p>Thank you for publishing your work with us.</p>",
+  "buttonCollection": {
+    "centered": true,
+    "buttons": [
+      {
+        "text": "Download A4",
+        "classes": "button--default",
+        "path": "#"
```
[the remark this answers is not in the document]
Comment (thewilkybarkid) [reply source/_patterns/01-molecules/components/download.json]: Yeah, inconsistent. Outside of the scope of this (in the Button pattern).

**Comment by giorgiosironi** [comment]
At https://s3.amazonaws.com/ci-pattern-library/814/?p=molecules-download I see the buttons on the same row, not stacked in a column, is that expected?

**Comment by thewilkybarkid** [comment]
Yep, spec in ticket.

File: source/_patterns/01-molecules/components/download.mustache [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    <picture>
+      <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/download.svg{{/assetRewrite}}"
+              type="image/svg+xml">
+      <img srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/download_2x.png{{/assetRewrite}} 276w"
+           src="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/download_1x.png{{/assetRewrite}}" sizes="138px"
+           alt="" class="download__icon">
+    </picture>
+
+    {{{text}}}
```
Comment (thewilkybarkid) [inline source/_patterns/01-molecules/components/download.mustache]: Maybe should be a list of paragraphs? (Would we ever want anything else here?)

**Comment by thewilkybarkid** [comment]
Trying to think of better names, `DownloadOptions`?

**Comment by giorgiosironi** [comment]
`DownloadAlternatives` but the pattern itself doesn't require that the available files are alternative to each other.
`DownloadAssets` (seen at https://github.com/elifesciences/proofreader-python/releases) is too overloaded and only refers to the files.
For comparison, we have an `ArticleDownloadLinksList` in the article page. Given the hardcoded icon it could conceivably be called `Personalis|edCoverDownload` until something more general will be extracted, for the benefit of the next person trying to find where this pattern lives.

**Comment by thewilkybarkid** [comment]
Happy to go for a specific name for now. If we need to reuse it elsewhere we can make it more reusable.

**Comment by thewilkybarkid** [comment]
Though I'll use a real 'l'. 😉

**Comment by giorgiosironi** [comment]
(was meant to be `personali(s|z)ed`)

**Pull request merged**